queries.sql

```diff
@@ -49,4 +49,47 @@ SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg) AS min_weight_kg, MAX(weight_kg) AS max_weight_kg  FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) AS avg_escape_attempts  FROM animals WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31' GROUP BY species;
 
+/* Day 3 query multiple tables */
 
+/* What animals belong to Melody Pond? */
+SELECT name FROM animals A
+    JOIN owners O
+    ON A.owner_id = O.id
+    WHERE O.full_name = 'Melody Pond';
+
+/* List of all animals that are pokemon (their type is Pokemon). */
+SELECT A.name FROM animals A
+    JOIN species S
+    ON A.species_id = S.id
+    WHERE S.name = 'Pokemon';
+
+/* List all owners and their animals, remember to include those that don't own any animal. */
+SELECT full_name, name FROM owners O
+    LEFT JOIN animals A
+    ON O.id = A.owner_id;
+
+/* How many animals are there per species? */
+SELECT S.name, COUNT(A.name) FROM species S
+    LEFT JOIN animals A
+    ON S.id = a.species_id
+    GROUP BY s.name;
+
+/* List all Digimon owned by Jennifer Orwell. */
+SELECT A.name FROM owners O
+    LEFT JOIN animals A
+    ON O.id = A.owner_id
+    LEFT JOIN species S
+    ON A.species_id = S.id
+    WHERE full_name = 'Jennifer Orwell' AND S.name = 'Digimon';
+
+/* List all animals owned by  that haven't tried to escape. */
+SELECT name FROM owners O
+    JOIN animals A
+    ON O.id = A.owner_id
+    WHERE O.full_name = 'Dean Winchester' AND escape_attempts = 0;
+
+/* Who owns the most animals? */
+SELECT full_name, COUNT(name) FROM owners O
+    LEFT JOIN animals A
+    ON O.id = A.owner_id
+    GROUP BY full_name;
```

Comment:
- Great queries so far. 👍 However, the last query says; `Who owns the most animals?`.
After running your query, this 👇 is what I got;

![](https://i.imgur.com/Eq3heew.png)

To get the person who owns the most animals, I believe the query should be written this 👇 way;

```
SELECT full_name, COUNT(name) FROM owners O
LEFT JOIN animals A
ON O.id = A.owner_id   GROUP BY full_name 
ORDER BY COUNT(name) 
DESC LIMIT 1;
```
😉